src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_config.json

```diff
@@ -19,16 +24,23 @@
         "AWS::DynamoDB::Table",
         "AWS::EC2::CustomerGateway",
         "AWS::EC2::EIP",
+        "AWS::EC2::EgressOnlyInternetGateway",
+        "AWS::EC2::FlowLog",
         "AWS::EC2::Host",
         "AWS::EC2::Instance",
         "AWS::EC2::InternetGateway",
+        "AWS::EC2::NatGateway",
         "AWS::EC2::NetworkAcl",
         "AWS::EC2::NetworkInterface",
+        "AWS::EC2::RegisteredHAInstance",
```

Comment:
I can't seem to find this one...

Reply:
Right. A number of them are not there, namely:

* AWS::Config::ResourceCompliance
* AWS::EC2::RegisteredHAInstance
* AWS::WAFv2::IPSet
* AWS::WAFv2::ManagedRuleSet
* AWS::WAFv2::RegexPatternSet
* AWS::WAFv2::RuleGroup
* AWS::WAFv2::WebACL

However, they:
1. exist in the web console
2. are handled correctly by CloudFormation deployments

This is my first ever contribution to OSS. Please advise on what would be an acceptable proof of correctness here.